src/lib/util/options.ml

```diff
@@ -346,13 +346,15 @@ let get_timelimit_per_goal () = !timelimit_per_goal
 (** Output options *)
 
 let interpretation = ref INone
+let print_all_interpretations = ref false
```

Comment:
Since we are now using the more standard "model" instead of "interpretations", this probably should be `print_all_models` instead.

(As an aside, I believe that both CVC5 and Z3 call this option `dump_models` and maybe we should do the same)